<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberPanel</class>
 <widget class="QDialog" name="MemberPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="introLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>491</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kenyan Coffee</family>
     <pointsize>45</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Member's Dashboard</string>
   </property>
  </widget>
  <widget class="QLabel" name="thLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Neou</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Total Hours:</string>
   </property>
  </widget>
  <widget class="QLabel" name="hourLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kenyan Coffee</family>
     <pointsize>45</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QListWidget" name="eventList">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>481</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="logButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="printButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
